### DISCUSSION TITLE
Home Assistant OS 12.2.rc1

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3276] [Announcements]
## Home Assistant Operating System

* kernel: enable MPTCP support (#3248) @matttbe
* Update Buildroot to 2024.02 (#3241) @sairon
* Bump buildroot to 2024.02.1 (#3268) @sairon
* Bump buildroot to update Docker to v25.0.5 (#3271) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.23 (#3270) @sairon

## Open Virtual Appliance

* Linux: Update kernel to 6.6.23 (#3270) @sairon

## Generic x86-64

* Linux: Update kernel to 6.6.23 (#3270) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.23 (#3270) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.23 (#3270) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.23 (#3270) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.23 (#3270) @sairon

## Build System

* Bump docker/setup-buildx-action from 3.1.0 to 3.2.0 (#3257) @dependabot
* Bump docker/build-push-action from 5.2.0 to 5.3.0 (#3256) @dependabot
* Bump docker/login-action from 3.0.0 to 3.1.0 (#3258) @dependabot

## Documentation

* List Nabu Casa appliances under boards README.md (#3260) @Hedda

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/12.2.rc1'>Home Assistant OS 12.2.rc1</a>.</em>

**Comment by rononline** [comment]
I can't login via ssh anymore:

```
❯ ssh hassio@homeassistant.local
sudo: unable to allocate pty: Read-only file system
Connection to homeassistant.local closed.
```

**Reply by rononline** [reply]
this is what I see in the logging:

```
Connection from 192.1.1.29 port 61663 on 192.1.1.98 port 22 rdomain ""
Accepted key RSA SHA256:1XXXXXXXXXXXXXXXX5kXYk5mK+atg found at /etc/ssh/authorized_keys:1
Postponed publickey for hassio from 192.1.1.29 port 61663 ssh2 [preauth]
Accepted key RSA SHA256:1XXXXXXXXXXXXXXXX5kXYk5mK+atg found at /etc/ssh/authorized_keys:1
Accepted publickey for hassio from 192.1.1.29 port 61663 ssh2: RSA SHA256:1XXXXXXXXXXXXXXXX5kXYk5mK+atg
User child is on pid 7120
Starting session: shell on pts/0 for hassio from 192.1.1.29 port 61663 id 0
/dev/pts/0: Permission denied
Received disconnect from 192.1.1.29 port 61663:11: disconnected by user
Disconnected from user hassio 192.1.1.29 port 61663
```

**Comment by hermanops** [comment]
The Yellow didn't get any  (kernel) updates?
